### ISSUE TITLE
Suggestion:Add an option to specify language downloaded.

### ISSUE THREAD

**Issue by yw981** [coursera-dl/coursera-dl#596]
🚨Please review the [Troubleshooting](../#troubleshooting) section
before reporting any issue. Don't forget to check also the current issues to
avoid duplicates.

## Add an option to specify language downloaded.
All of the subtitles of all languages are downloaded in current version. Could you add an option like "-l" to let users specify which language they want to download in the future version ?

### Your environment
* Operating System (name/version):Windows 10
* Python version:3.5
* coursera-dl version:

**Comment by balta2ar** [comment]
After merging this pull request https://github.com/coursera-dl/coursera-dl/pull/593 into master, I think you can download only English materials as follows: `-f "en.txt en.srt pdf mp4"`. But it's not in pip yet.
You see, having a ton of options is a pain of maintaining them whenever the code changes. So usually there has to be a very strong reason to add an option.

**Comment by dzhuang** [comment]
Now you can use the git version, to specify the languages to be downloaded, e.g., 

```
-sl en  # for English only, and 
-sl "en,de"  # for English and German, without spaces
-sl "en, de|fr"  # for English and German, if German not available, French will be downloaded, needed to be wrapped by quotes
```

If all specified langs are not available, then English subtitles/transcripts will be downloaded, if available.

**Comment by dzhuang** [comment]
I think this issue can be closed. @balta2ar

**Comment by balta2ar** [comment]
Thanks. Available in version 0.9.0. Closing.
